packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -206,7 +209,42 @@ class PreviewCommand extends ShorebirdCommand
   }
 
   Future<int> installAndLaunchIos(String appId, Release release) async {
-    return ExitCode.unavailable.code;
+    const platform = ReleasePlatform.ios;
+    final previewDirectory = cache.getPreviewDirectory(appId);
+    final runnerPath = p.join(
+      previewDirectory.path,
+      '${platform.name}_${release.version}.app',
```

Comment:
You repeat '${platform.name}_${release.version}' in this file, which is fine, but you might want a function 🤷

Reply:
Where is this repeated? Did you mean one in iOS and one in Android?